mountpoint-s3-crt/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.6.3 (April 10, 2024)
+
+* Update to latest CRT dependencies
+* Adopt new async write API for PutObject requests ([#832](https://github.com/awslabs/mountpoint-s3/pull/832))
```

Comment:
Should this be 0.7.0 with the new async write API? it seems like a breaking change for the crt crate.

same for `mountpoint-s3-crt-sys` crate, I think we want them to be consistent.